hidden/AppDelegate.swift

```diff
@@ -65,5 +66,11 @@ class AppDelegate: NSObject, NSApplicationDelegate{
         return true
     }
     
+    func grantAccess() {
+        if !GlobalEventMoniter.grantAccess() {
+            // TODO: guide user to enable accessibility access
+            print("Accessibility access not granted")
```

Comment:
Does this violate Appstore policy? If it does we can't use it. If it doesn't we need UI for this.

Reply:
Well, I've tried not granting accessibility permissions to the app, but everything seem to be working fine to me 😂. Maybe we doesn't need accessibility permissions at all.